src/py/flwr/server/strategy/dp_fixed_clipping.py

```diff
@@ -220,7 +220,7 @@ class DifferentialPrivacyClientSideFixedClipping(Strategy):
 
     >>> strategy = fl.server.strategy.FedAvg(...)
 
-    Wrap the strategy with the `DifferentialPrivacyServerSideFixedClipping` wrapper:
+    Wrap the strategy with the `DifferentialPrivacyClientSideFixedClipping` wrapper:
```

Comment:
Is this to fix a typo? Or the name of the strategy wrapper is changed?

Reply:
this is to fix a typo